content/applications/websites/ecommerce/managing_products/variants.rst

```diff
@@ -1,110 +1,88 @@
-=======================
-Manage product variants
-=======================
+================
+Product variants
+================
 
-Product variants are used to offer variations of the same product to your customers on the product's
-page.
+Product variants are variations such as color, material, etc. of a same product. These variations
+can differ in price and availability from the product. Product variants can either be
+:doc:`created <../../../sales/sales/products_prices/products/variants>` or
+:doc:`imported <../../../sales/sales/products_prices/products/import>`.
 
-Let's say one of your customers selects a T-shirt from your product catalog.
-
-Then, they choose the size and color they want. The various size and color combinations are known as
-product variants.
-
-Create attributes & variants
-============================
-
-In order to create various attributes and variants for your products, you first need to turn on
-*Variants* in :menuselection:`Sales --> Configuration --> Settings`.
-
-Then, select a product from the Products list (:menuselection:`Sales / Website --> Products -->
-Products`). When the product detail form opens, click on *Edit*, and go to the *Variants* tab.
-
-Here, you can add as many different attributes as you'd like by clicking on *Add a line*.
-
-These attributes appear as three different types:
+.. note::
+   :guilabel:`Product Variants` has to be enabled under :menuselection:`Configuration --> Settings`,
+   in the :guilabel:`Shop - Products` section.
```

Comment:
```suggestion
   :guilabel:`Product Variants` have to be enabled under :menuselection:`Configuration --> Settings`,
   in the :guilabel:`Shop - Products` section.
```

I know it's "the option" that has to be enabled, but it sounds weird when the words before the verb are in the plural form